docs/source/_toctree.yml

```
@@ -78,3 +78,17 @@
       title: Optimization
     title: Reference
   title: Torch FX
+- sections:
+  - local: bettertransformer/overview
+    title: Overview
+  - local: bettertransformer/quickstart
+    title: Quickstart
+  - sections:
+    - local: bettertransformer/tutorials/content
+      title: Content
+    title: Tutorials
+  - sections:
+    - local: bettertransformer/contribution/content
+      title: Content
+    title: Contribute
+  title: BetterTransformer integration
```

```suggestion
- sections:
  - local: bettertransformer/overview
    title: Overview
  - sections:
    - local: bettertransformer/tutorials/convert
      title: Convert Transformers models to use BetterTransformer
    - local: bettertransformer/tutorials/contribute
      title: How to add support for new architectures?
    title: Tutorials
  title: BetterTransformer integration
```

Maybe something like this? and refactor the `tutorials/`, `contribution/` accordingly?

---

Thanks! Fixed in 4abd203